changelogs/fragments/4578-ipa_dnsrecord-add_multiple_record_support.yml

```diff
@@ -0,0 +1,7 @@
+---
+breaking_changes:
+ - ``record_value`` - an value with "," inside is interpreated by Ansible to split to array, so if your e.g. TXT-Record contains comma use an array with one value
```

Comment:
If this is a breaking change, it will never get merged. So make it non-breaking.
```suggestion
```